src/osg/Texture1D.cpp

```diff
@@ -211,7 +211,7 @@ void Texture1D::apply(State& state) const
 
         applyTexParameters(GL_TEXTURE_1D,state);
 
-        // update the modified count to show that it is up to date.
```

Comment:
Where did these changes come from? I am not seeing them in makOsg 3.4

Reply:
This comment refers to the whole file.

Reply:
Nevermind, I see that it is because selectSizedInternalFormat was removed from texture.cpp. I think now that I have a new version of texture.cpp specified in the comments for it then we can put these changes back in.